appservice/src/main/java/nu/yona/server/analysis/rest/ActivityControllerBase.java

```diff
@@ -79,12 +81,14 @@
 	protected static final int WEEKS_DEFAULT_PAGE_SIZE = 2;
 	protected static final int DAYS_DEFAULT_PAGE_SIZE = 3;
 	protected static final int MESSAGES_DEFAULT_PAGE_SIZE = 4;
-	protected static final String PREV_REL = "prev"; // IANA reserved, so will not be prefixed
```

Comment:
I don't think there are many. This comment was wrapped in an earlier formatting action (with a different formatter) and not joined again when reformatted with the Yona formatter. Corrected that and also the next one.

Besides this, I see a line wrapping change, but that was originally too long and now corrected.